modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/workflow/mgt/WorkflowManagementTestCase.java

```diff
@@ -309,8 +292,760 @@ public void testAssociationForNonMatch() {
+        } catch (Exception e) {
+            Assert.fail("failed to add updateUserListOfRole workflow.");
+        }
+        try {
+            usmClient.updateUserListOfRole(roleName2, new String[]{userName3}, new String[]{});
+        } catch (Exception e) {
+            Assert.fail("Failed to update user list of role.");
+        }
+        try {
+            usmClient.deleteUser(userName3);
+            Assert.fail("Should throw an exception since there is a pending updateUserListOfRole workflow with this " +
+                    "user.");
+        } catch (Exception e) {
+            //test pass
+        }
+
+        try {
+            AssociationDTO[] associations = client.listAssociationsForWorkflow(workflowId);
+            for (AssociationDTO association : associations) {
+                if ("TestUpdateUserListOfRoleAssociation".equals(association.getAssociationName())) {
+                    associationId = association.getAssociationId();
+                    client.deleteAssociation(associationId);
+                    break;
+                }
+            }
+        } catch (Exception e) {
+            log.error("Error while deleting updateUserListOfRole association at testAddUserOperation.");
+        }
+
+        try {
+            client.addAssociation(workflowId, "TestUpdateRoleListOfUserAssociation", WorkflowConstants
+                    .UPDATE_USER_ROLES_EVENT, "boolean(1)");
+        } catch (Exception e) {
+            Assert.fail("failed to add updateRoleListOfUser workflow.");
+        }
+        try {
+            usmClient.updateRoleListOfUser(userName4, new String[]{roleName2}, new String[]{});
+        } catch (Exception e) {
+            Assert.fail("Failed to update user list of role.");
+        }
+        try {
+            usmClient.deleteUser(userName4);
+            Assert.fail("Should throw an exception since there is a pending updateRoleListOfUser workflow with this " +
+                    "user.");
+        } catch (Exception e) {
+            //test pass
+        }
+
+        try {
+            AssociationDTO[] associations = client.listAssociationsForWorkflow(workflowId);
+            for (AssociationDTO association : associations) {
+                if ("TestUpdateRoleListOfUserAssociation".equals(association.getAssociationName())) {
+                    associationId = association.getAssociationId();
+                    client.deleteAssociation(associationId);
+                    break;
+                }
+            }
+        } catch (Exception e) {
+            log.error("Error while deleting updateRoleListOfUser association at testAddUserOperation.");
+        }
+
+        try {
+            client.addAssociation(workflowId, "TestDeleteClaimOfUserAssociation", WorkflowConstants
+                    .DELETE_USER_CLAIM_EVENT, "boolean(1)");
+        } catch (Exception e) {
+            Assert.fail("failed to add deleteClaimOfUser workflow.");
```

Comment:
no need to catch and assert the exception. just add to method signature. please fix in all places